app/models/concerns/import_utils.rb

```diff
@@ -102,6 +103,12 @@ def remove_feedburner_tracker(str)
     str.sub(regex, "").strip
   end
 
+  def remove_podcastchoices_link(str)
+    if str.present?
+      str.sub(/(<p>)?learn more about your ad choices.+podcastchoices.com\/adchoices\.?(<\/a>)?(<\/p>)?/i, "").strip
```

Comment:
Bit magical, but this regex will strip off both the HTML and plain-text version of these sentences:

```
Learn more about your ad choices. Visit podcastchoices.com/adchoices

<p>Learn more about your ad choices. Visit <a href="https://podcastchoices.com/adchoices">podcastchoices.com/adchoices</a></p>
```

And there are tests!

Reply:
where does this text come from? it seems specific.

Reply:
It's on all megaphone generated RSS (apparently).  And that url has very megaphone/spotify specific language.